### PR TITLE
[15.0][IMP] account_banking_mandate: Set sequence when creating a mandate with empty mandate reference

### PR DESCRIPTION
**Pull request by victoralmau**
Set sequence when creating a mandate with empty mandate reference

Please @pedrobaeza and @carolinafernandez-tecnativa can you review it?

@Tecnativa TT50317

### CODE REVIEW

File: account_banking_mandate/models/account_banking_mandate.py [hunk 1]
```diff
@@ -176,8 +176,10 @@ def _check_valid_state(self):
     @api.model_create_multi
     def create(self, vals_list):
         for vals in vals_list:
-            unique_mandate_reference = vals.get("unique_mandate_reference", "/")
-            if unique_mandate_reference == "/":
+            if (
+                not vals.get("unique_mandate_reference")
+                or vals.get("unique_mandate_reference") == "/"
+            ):
```
Comment (pedrobaeza) [inline account_banking_mandate/models/account_banking_mandate.py]:
```suggestion
            if vals.get("unique_mandate_reference", "/") == "/":
```

Comment (victoralmau) [reply account_banking_mandate/models/account_banking_mandate.py]: Sorry, but that's how it was previously and it didn't cover the use case of leaving the reference empty.

Comment (pedrobaeza) [reply account_banking_mandate/models/account_banking_mandate.py]: Ah, OK, that's right. Then it should be:

```suggestion
            if (vals.get("unique_mandate_reference") or "/") == "/":
```

**Review by pedrobaeza — APPROVED** [review]

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :)

**Pull request merged**